src/zzz_od/application/random_play/random_play_app.py

```
@@ -139,24 +139,62 @@ def choose_promoter(self) -> OperationRoundResult:
+            time.sleep(0.5)
+            return self.round_by_find_and_click_area(self.last_screenshot, '影像店营业', '确认', success_wait=1, retry_wait=1)
+
+        log.info('备用代理人匹配也失败: %s' % backup_agent)
+
+        # 识别区域中的第一个代理人
+        part = cv2_utils.crop_image_only(self.last_screenshot, area.rect)
+        ocr_results = self.ctx.ocr.run_ocr(part)
+
+        if ocr_results:
+            # 获取第一个识别结果的位置
+            first_ocr = list(ocr_results.items())[0]
```

Ruff (0.15.2)

[warning] Prefer `next(iter(ocr_results.items()))` over single element slice

Replace with `next(iter(ocr_results.items()))`

(RUF015)